init-root.sh

```diff
@@ -146,45 +146,45 @@ cudos-noded add-genesis-account $FAUCET_ADDRESS "1000000000000000000000000${BOND
+ACCOUNT5_ADDRESS=$(  cudos-noded keys show account5 -a  )
+cudos-noded add-genesis-account $ACCOUNT5_ADDRESS "1000000000000000000${BOND_DENOM}"
 
-echo "write senior detail business try dirt sort rural donate way acid flame" | cudos-noded keys add account_6 --recover
-ACCOUNT_6_ADDRESS=$(  cudos-noded keys show account_6 -a  )
-cudos-noded add-genesis-account $ACCOUNT_6_ADDRESS "1000000000000000000${BOND_DENOM}"
+echo "write senior detail business try dirt sort rural donate way acid flame" | cudos-noded keys add account6 --recover
+ACCOUNT6_ADDRESS=$(  cudos-noded keys show account6 -a  )
+cudos-noded add-genesis-account $ACCOUNT6_ADDRESS "1000000000000000000${BOND_DENOM}"
 
-echo "debris beef trip coral victory boy picture lab best royal wrestle path" | cudos-noded keys add account_7 --recover
-ACCOUNT_7_ADDRESS=$(  cudos-noded keys show account_7 -a  )
-cudos-noded add-genesis-account $ACCOUNT_7_ADDRESS "1000000000000000000${BOND_DENOM}"
+echo "debris beef trip coral victory boy picture lab best royal wrestle path" | cudos-noded keys add account7 --recover
+ACCOUNT7_ADDRESS=$(  cudos-noded keys show account7 -a  )
+cudos-noded add-genesis-account $ACCOUNT7_ADDRESS "1000000000000000000${BOND_DENOM}"
 
-echo "damp bunker feed quit maze execute raccoon office error squirrel believe measure" | cudos-noded keys add account_8 --recover
-ACCOUNT_8_ADDRESS=$(  cudos-noded keys show account_8 -a  )
-cudos-noded add-genesis-account $ACCOUNT_8_ADDRESS "1000000000000000000${BOND_DENOM}"
+echo "damp bunker feed quit maze execute raccoon office error squirrel believe measure" | cudos-noded keys add account8 --recover
+ACCOUNT8_ADDRESS=$(  cudos-noded keys show account8 -a  )
+cudos-noded add-genesis-account $ACCOUNT8_ADDRESS "1000000000000000000${BOND_DENOM}"
 
-echo "sea prison zero sugar episode meadow neutral silly chronic creek danger flat" | cudos-noded keys add account_9 --recover
-ACCOUNT_9_ADDRESS=$(  cudos-noded keys show account_9 -a  )
-cudos-noded add-genesis-account $ACCOUNT_9_ADDRESS "1000000000000000000${BOND_DENOM}"
+echo "sea prison zero sugar episode meadow neutral silly chronic creek danger flat" | cudos-noded keys add account9 --recover
+ACCOUNT9_ADDRESS=$(  cudos-noded keys show account9 -a  )
+cudos-noded add-genesis-account $ACCOUNT9_ADDRESS "1000000000000000000${BOND_DENOM}"
 
-echo "save wear nature point april cage hybrid peasant penalty silver sadness friend" | cudos-noded keys add account_10 --recover
-ACCOUNT_10_ADDRESS=$(  cudos-noded keys show account_10 -a  )
-cudos-noded add-genesis-account $ACCOUNT_10_ADDRESS "1000000000000000000${BOND_DENOM}"
+echo "save wear nature point april cage hybrid peasant penalty silver sadness friend" | cudos-noded keys add account10 --recover
+ACCOUNT10_ADDRESS=$(  cudos-noded keys show account10 -a  )
+cudos-noded add-genesis-account $ACCOUNT10_ADDRESS "1000000000000000000${BOND_DENOM}"
 
```

Comment:
I am not sure if we need to include these changes in this pr, but if we had to we have to change the default deployer account name in the contract-utils.js as well. Also we need only one init-root.js

Reply:
should we move the file to the config folder?

Reply:
OK, I will leave root file unchanged. If removed without further changes, it will break node command. The file should be removed from root with `node` command refactoring.
If you prefer I can leave the changes for node refactoring or separate task.